src/htm/algorithms/SpatialPooler.hpp

```diff
@@ -148,20 +156,20 @@ class SpatialPooler : public Serializable
         columns the same regardless of the size of their receptive
         fields.
 	If numActivePerInhArea is specified then 
-	localAreaDensity must be < 0, and vice versa.
+	@ref localAreaDensity must be < 0, and vice versa.
 
-  @param stimulusThreshold This is a number specifying the minimum
+  @param stimulusThreshold This is a number specifying the minimum //TODO replace with `robustness` 0..1.0, which will affect this & synPermInc/Dec
```

Comment:
stimulusThreshold well represents "robustness" (to noise) of the segment. 

- bump default not to be too small (to 2,3,4,..?)
- must not be too high, or no segment will be able to satisfy and no learning will occur -> auto check that `num potential synapses on segment is x-times (2 times ?) bigger than the threshold`
- in "smart" replace with "robustness" [0.0..1.0]